README.md

```diff
@@ -2,31 +2,818 @@
+unique values within a column.
+
+
+<table>
+<tr>
+<th>
+Query
+</th>
+<th>
+Result
+</th>
+</tr>
+
+<tr>
+<td>
+Display all columns and rows:
+
+```sql
+SELECT *
+FROM pet;
+```
+
+</td>
+<td>
+
+```text
+id  name        species	breed	            age
+
+1   Moe         cat     Tabby	            10
+2   Hana        cat     Tabby	            1
+3   Lil' Bub    cat     American Shorthair  5
+4   Snuggles    dog     Bichon	            5
+5   Honey       dog     Cavachon            NULL
+6   Herbie      fish    NULL                4
+7   Fifi        dog     Poodle              1
+8   Lulu        dog     Yorkiepoo           12
+```
+
+</td>
+</tr>
+
+
+
+<tr>
+<td>
+Display a subset of columns:
+
+```sql
+SELECT name, species
+FROM pet;
+```
+
+</td>
+<td>
+
+```text
+name        species
+
+Moe         cat
+Hana        cat
+Lil' Bub    cat
+Snuggles    dog
+Honey       dog
+Herbie      fish
+Fifi        dog
+Lulu        dog
+```
+
+</td>
+</tr>
+
+<tr>
+<td>
+Display unique values within a column:
+
+```sql
+SELECT DISTINCT species
+FROM pet; 
+```
+
+</td>
+<td>
+
+```text
+species
+
+dog
+cat
+fish
+```
+
+</td>
+</tr>
+
+</table>
+
+
+### WHERE clause
+
+The `WHERE` clause is used to retrieve a subset of rows
+that match a given predicate.  The predicate is a boolean
+expression that compares column values using:
+
+- Relational operators `=`, `!=`, `<`, `<=`, `>`, `>=`
+- Boolean operators `AND`, `OR`, `NOT`
+- Shorthand boolean operators `IN`, `NOT IN`
+- Shorthand range operator `BETWEEN`
+- Null operator `IS NULL` and `IS NOT NULL`
+- Pattern matching operator `LIKE`
+
+<table>
+<tr>
+<th>
+Query
+</th>
+<th>
+Result
+</th>
+</tr>
+
+<tr>
+<td>
+Display name and age of pets younger than 5:
+
+```sql
+SELECT name, age
+FROM pet
+WHERE age < 5;
+```
+
+</td>
+<td>
+
+```text
+name    age
+
+Hana    1
+Herbie  4
+Fifi    1
+```
+
+</td>
+</tr>
+
+
+
+<tr>
+<td>
+Display name, breed, age of all cats:
+
+```sql
+SELECT name, breed, age
+FROM pet
+WHERE species = 'cat';
+```
+
+Notice SQL uses only one equal sign to
+test for equality.
+
+</td>
+<td>
+
+```text
+name        breed               age
+
+Moe         Tabby               10
+Hana        Tabby               1
+Lil' Bub    American Shorthair  5
+```
+
+</td>
+</tr>
+
+
+<tr>
+<td>
+Display name, species of any
+pet that is not a cat:
+
+```sql
+SELECT name, species
+FROM pet
+WHERE species != 'cat';
+```
+
+</td>
+<td>
+
+```text
+name        species
+
+Snuggles    dog         
+Honey       dog
+Herbie      fish
+Fifi        dog
+Lulu        dog   
+```
+
+</td>
+</tr>
+
+
+<tr>
+<td>
+Display name, breed, and age of
+cats younger than 5:
+
+```sql
+SELECT name, breed, age
+FROM pet
+WHERE species = 'cat' AND age < 5;
+```
+
+</td>
+<td>
+
+```text
+name        breed               age
+
+Hana         Tabby               1
+```
+
+</td>
+</tr>
+
+
+
+<tr>
+<td>
+Display name and age of
+pets between 4 and 10 years old (inclusive):
+
+```sql
+SELECT name, breed, age
```

Comment:
Remove breed from the query to match the instruction and output.

Reply:
fixed